<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIFavoriteHubEditor</class>
 <widget class="QDialog" name="UIFavoriteHubEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Hub</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_AUTOCONNECT">
        <property name="text">
         <string>Autoconnect on startup</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_NAME"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_ADDRESS"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_DESC"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_ENC"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Encoding</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBox_CID">
        <property name="text">
         <string>Client ID</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="checkBox_IP">
        <property name="text">
         <string>External IP</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox_CID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>EiskaltDC++ V:2.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>++ V:0.75 </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>StrgDC++ V:2.30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ApexDC++ V: 1.2.2 </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FlylinkDC++ V:(r500) </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HomeDC++ V:2.22 </string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_IP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMask">
         <string>000.000.000.000; </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="checkBox_USEINTERNET">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Auto update from Internet</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_DISABLECHAT">
        <property name="text">
         <string>Disable chat</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Identification</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_NICK">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_PASSWORD">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_USERDESC">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_NICK">
        <property name="text">
         <string>Nick</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_USERDESC">
        <property name="text">
         <string>User description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UIFavoriteHubEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UIFavoriteHubEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_NICK</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_NICK</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_USERDESC</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_USERDESC</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_CID</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_CID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_IP</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_IP</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_IP</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_USEINTERNET</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
